packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-step-checker/ola-step-checker.controller.js

```diff
@@ -49,4 +29,12 @@ export default class {
       maximumSignificantDigits: 1,
     }).format(price);
   }
+
+  hasBandwidthExtension() {
+    // @TODO only for infrav2 and hgv2
```

Comment:
If there is still a `TODO` maybe the PR should be a draft ?

Reply:
Yes, I think next lines are in draft :)

Reply:
i put this PR in blocked because of that & i wanted a review
but yes draft is fine as long as we are reviewing drafts :)